onnx/defs/quantization/defs.cc

```diff
@@ -27,8 +28,9 @@ ONNX_OPERATOR_SET_SCHEMA(
         .Input(
             1,
             "y_scale",
-            "Scale for doing quantization to get 'y'. It can be a scalar, which means per-tensor/layer quantization, "
-            "or a 1-D Tensor for per-axis quantization.",
+            "Scale for doing quantization to get 'y'. For per-tensor/layer quantization the scale is a scalar, for "
+            "per-axis quantization it is a 1-D Tensor and for blocked quantization it has the same shape as the "
+            "input, except for one dimension in which blocking is performed.",
```

Comment:
`except for one dimension...` which attribute indicates that dimension? axis does not seem to used in this case according to the documentation.

Reply:
You can move this information in the main block of the documentation and refer to it if you prefer. The documentation often lacks precision and it happens that it needs to be completed because the behaviour is ambiguous for one particular case. I would say, the more precise it is, the better it is. +@gramalingam

Reply:
I've added commit #45c446a, rewriting the quantization operator definition, rather than patching it. I hope it is clearer now.
I've used the term "quantization granularity" to distinguish between per-tensor/per-axis/blocked quantization.
I've considered the more generic "quantization mode", but opted for the more informative term.
Minor modifications were made to the dequantization definition, to avoid repetition.

Please don't hesitate to suggest improvements.

Reply:
@xadupre please take a look and LMK if you have any additional comments.